armory/eval/evaluator.py

```diff
@@ -92,10 +92,28 @@ def _gather_env_variables(self):
         if not self.armory_global_config["verify_ssl"]:
             self.extra_env_vars["VERIFY_SSL"] = "false"
 
+        cuda_var_exists = (
+            True if os.getenv("CUDA_VISIBLE_DEVICES") is not None else False
+        )
+        if cuda_var_exists and self.no_docker:
+            log.warning(
+                "CUDA_VISIBLE_DEVICES is set; any Armoy gpu instructions will be ignored"
+            )
+            # Existing value should override armory flags
```

Comment:
Very minor formatting change
```suggestion
            # Existing value should override armory flags
            log.warning(
                "CUDA_VISIBLE_DEVICES is set; any Armoy gpu instructions will be ignored"
            )
```